pkgs/build-support/node/fetch-yarn-deps/index.js

```diff
@@ -37,7 +37,9 @@ const downloadFileHttps = (fileName, url, expectedHash, hashType = 'sha1') => {
 			res.on('end', () => {
 				file.close()
 				const h = hash.read()
-				if (h != expectedHash) return reject(new Error(`hash mismatch, expected ${expectedHash}, got ${h}`))
+				if (expectedHash === undefined){
```

Comment:
At first glance this looks fine to include in this PR, but could you put it in a separate commit and explain the motivation in the message? This way the change could be easily reverted if it would end up causing trouble.

Reply:
@erictapen thank you for calling this out. I'll get the commit separated 👍🏼

Reply:
Are you sure you also use the latest revision? Your [flake.lock](https://gitlab.com/tootcat/nixos-config/-/blob/79890506fb038a7727d6b9e4e2538314ab096411/flake.lock) doesn't point to the current commit. You might need to run `nix flake update --commit-lock-file` first.